src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -205,6 +205,12 @@ p5.RendererGL.prototype.endShape = function(
   this._processVertices(...arguments);
   this.isProcessingVertices = false;
 
+  // Check the number of vertices and if equal to 3
+  // Change the shapemode to traingle
```

Comment:
It would be helpful to explain *why* we do this, since currently this comment just mirrors the exact logic of the code below. It would me more helpful, I think, if we explain that we do this for speed: normally we break bigger shapes up into triangles, but we can skip this and get a bit of a performance boost when we know the shape is already just a triangle.